### ISSUE TITLE
RyuJIT/x86: two tests time out under GCStress=0xC, JITStress=1

### ISSUE THREAD

**Issue by pgavlin** [dotnet/coreclr#7695]
e.g. https://ci.dot.net/job/dotnet_coreclr/job/master/job/jitstress/job/x86_ryujit_checked_windows_nt_gcstress0xc_jitstress1/8/console

```
21:08:12 D:\j\workspace\x86_ryujit_ch---59c7f19c\bin\tests\Windows_NT.x86.Checked\TestWrappers\JIT\JIT.XUnitWrapper.cs(301359): error : JIT._opt_virtualstubdispatch_bigvtbl_bigvtbl_cs_d_bigvtbl_cs_d_._opt_virtualstubdispatch_bigvtbl_bigvtbl_cs_d_bigvtbl_cs_d_cmd [FAIL] [D:\j\workspace\x86_ryujit_ch---59c7f19c\tests\runtest.proj]
21:08:12         cmdLine:D:\j\workspace\x86_ryujit_ch---59c7f19c\bin\tests\Windows_NT.x86.Checked\JIT\opt\virtualstubdispatch\bigvtbl\bigvtbl_cs_d\bigvtbl_cs_d.cmd Timed Out
21:08:12         
21:08:12   
21:08:12   Return code:      -100
21:08:12   Raw output file:      D:\j\workspace\x86_ryujit_ch---59c7f19c\bin\tests\Windows_NT.x86.Checked\Reports\JIT\opt\virtualstubdispatch\bigvtbl\bigvtbl_cs_d\bigvtbl_cs_d.output.txt
21:08:12   Raw output:
21:08:12   BEGIN EXECUTION
21:08:12          "D:\j\workspace\x86_ryujit_ch---59c7f19c\tests\..\bin\tests\Windows_NT.x86.Checked\Tests\Core_Root\corerun.exe" bigvtbl_cs_d.exe 
21:08:12         PASSED
21:08:12         Expected: 100
21:08:12         Actual: 100
21:08:12         END EXECUTION - PASSED
21:08:12         PASSED
21:08:12         cmdLine:D:\j\workspace\x86_ryujit_ch---59c7f19c\bin\tests\Windows_NT.x86.Checked\JIT\opt\virtualstubdispatch\bigvtbl\bigvtbl_cs_d\bigvtbl_cs_d.cmd Timed Out
21:08:12         Test Harness Exitcode is : -100
21:08:12         
21:08:12   To run the test:
21:08:12   > set CORE_ROOT=D:\j\workspace\x86_ryujit_ch---59c7f19c\bin\tests\Windows_NT.x86.Checked\Tests\Core_Root
21:08:12   > D:\j\workspace\x86_ryujit_ch---59c7f19c\bin\tests\Windows_NT.x86.Checked\JIT\opt\virtualstubdispatch\bigvtbl\bigvtbl_cs_d\bigvtbl_cs_d.cmd
21:08:12   
21:08:12         Expected: True
21:08:12         Actual:   False
21:08:12           D:\j\workspace\x86_ryujit_ch---59c7f19c\bin\tests\Windows_NT.x86.Checked\TestWrappers\JIT\JIT.XUnitWrapper.cs(301359,0): at JIT._opt_virtualstubdispatch_bigvtbl_bigvtbl_cs_d_bigvtbl_cs_d_._opt_virtualstubdispatch_bigvtbl_bigvtbl_cs_d_bigvtbl_cs_d_cmd()
22:14:58 D:\j\workspace\x86_ryujit_ch---59c7f19c\bin\tests\Windows_NT.x86.Checked\TestWrappers\JIT\JIT.XUnitWrapper.cs(301497): error : JIT._opt_virtualstubdispatch_bigvtbl_bigvtbl_cs_r_bigvtbl_cs_r_._opt_virtualstubdispatch_bigvtbl_bigvtbl_cs_r_bigvtbl_cs_r_cmd [FAIL] [D:\j\workspace\x86_ryujit_ch---59c7f19c\tests\runtest.proj]
22:14:58         cmdLine:D:\j\workspace\x86_ryujit_ch---59c7f19c\bin\tests\Windows_NT.x86.Checked\JIT\opt\virtualstubdispatch\bigvtbl\bigvtbl_cs_r\bigvtbl_cs_r.cmd Timed Out
22:14:58         
22:14:58   
22:14:58   Return code:      -100
22:14:58   Raw output file:      D:\j\workspace\x86_ryujit_ch---59c7f19c\bin\tests\Windows_NT.x86.Checked\Reports\JIT\opt\virtualstubdispatch\bigvtbl\bigvtbl_cs_r\bigvtbl_cs_r.output.txt
22:14:58   Raw output:
22:14:58   BEGIN EXECUTION
22:14:58          "D:\j\workspace\x86_ryujit_ch---59c7f19c\tests\..\bin\tests\Windows_NT.x86.Checked\Tests\Core_Root\corerun.exe" bigvtbl_cs_r.exe 
22:14:58         PASSED
22:14:58         Expected: 100
22:14:58         Actual: 100
22:14:58         END EXECUTION - PASSED
22:14:58         PASSED
22:14:58         cmdLine:D:\j\workspace\x86_ryujit_ch---59c7f19c\bin\tests\Windows_NT.x86.Checked\JIT\opt\virtualstubdispatch\bigvtbl\bigvtbl_cs_r\bigvtbl_cs_r.cmd Timed Out
22:14:58         Test Harness Exitcode is : -100
22:14:58         
22:14:58   To run the test:
22:14:58   > set CORE_ROOT=D:\j\workspace\x86_ryujit_ch---59c7f19c\bin\tests\Windows_NT.x86.Checked\Tests\Core_Root
22:14:58   > D:\j\workspace\x86_ryujit_ch---59c7f19c\bin\tests\Windows_NT.x86.Checked\JIT\opt\virtualstubdispatch\bigvtbl\bigvtbl_cs_r\bigvtbl_cs_r.cmd
22:14:58   
22:14:58         Expected: True
22:14:58         Actual:   False
22:14:58           D:\j\workspace\x86_ryujit_ch---59c7f19c\bin\tests\Windows_NT.x86.Checked\TestWrappers\JIT\JIT.XUnitWrapper.cs(301497,0): at JIT._opt_virtualstubdispatch_bigvtbl_bigvtbl_cs_r_bigvtbl_cs_r_._opt_virtualstubdispatch_bigvtbl_bigvtbl_cs_r_bigvtbl_cs_r_cmd()
```

We should consider disabling these tests after ensuring that the timeouts are benign.

**Comment by pgavlin** [comment]
These tests also time out under GCStress=0xC, JITStress=2: https://ci.dot.net/job/dotnet_coreclr/job/master/job/jitstress/job/x86_ryujit_checked_windows_nt_gcstress0xc_jitstress2/7/

**Comment by pgavlin** [comment]
One of these also times out under GCStress=0xC, ZapDisable=1: https://ci.dot.net/job/dotnet_coreclr/job/master/job/jitstress/job/x86_ryujit_checked_windows_nt_gcstress0xc_zapdisable/9/testReport/

**Comment by pgavlin** [comment]
And under GCStress=0xC, JitStress=2, ZapDisable=1: https://ci.dot.net/job/dotnet_coreclr/job/master/job/jitstress/job/x86_ryujit_checked_windows_nt_gcstress0xc_zapdisable_jitstress2/lastCompletedBuild/testReport/

**Comment by pgavlin** [comment]
This was fixed with #7739.
